packages/commons/tests/unit/awsSdkUtils.test.ts

```diff
@@ -88,6 +82,11 @@ describe('Helpers: awsSdk', () => {
     });
   });
 
+  it('concatenates the PT AWS_SDK_UA_APP_ID when one is already set', () => {
+    // Assess
+    expect(process.env.AWS_SDK_UA_APP_ID).toEqual('test/PT/TEST/2.25.1');
```

Comment:
This might fail when the version changes.
```suggestion
    expect(process.env.AWS_SDK_UA_APP_ID).toEqual(`test/PT/TEST/${version}`);
```